Library/ViewModels/CommentsViewModelTests.swift

```diff
@@ -503,8 +505,10 @@ internal final class CommentsViewModelTests: TestCase {
       self.configureFooterViewWithState.assertValues([.hidden])
 
       let updatedEnvelope = CommentsEnvelope.multipleCommentTemplate
+        |> \.slug .~ nil
+        |> \.updateID .~ "1"
```

Comment:
Just curious about why this test failed if this was omitted.

Reply:
Because if this has a slug it's seen as being a response from a project, not an update.